Main.java

```diff
@@ -9,7 +11,26 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    //
+    Scanner input = new Scanner(System.in);
+
+    //  Prompt the User to enter a temperature in celsius 
+    System.out.println("Please enter a temperature you would like to convert in degrees Celsius:");   
+
+    // take the input and store it in a variable 
+    int C = input.nextInt(); 
+
+    // Create a second variable that will preform the math formula 
+    int IE = (C*9)/5 + 32; 
+
+    // calculate the tempature in Fahrenheit 
+    int F = IE;
+
+    // Tell the user the answer in both Fahrenheit and Celsius 
+    System.out.println(C + "C is the same as " + F + "F"); 
```

Comment:
Good work! The comments are great and the math works just fine. The reason your test cases are failing is because the output is slightly different than what is shown in the sample. I believe the sample doesn't have the colon(:) and you might be missing a period in the final output. Also, the variable names C and F are poor names. Remember, variable names should always begin with a lowercase letter and describe what they are storing. A better name would be something like tempInC or degreeC.
Finally, you actually don't need that IE variable. You could just do this math right inside of F.
Otherwise, good work!